vars/scyllaCloudLongevityPipeline.groovy

```diff
@@ -0,0 +1,113 @@
+        }
+
+        stages {
+            stage('Checkout') {
+               steps {
+                  dir("siren-tests") {
+                    git(url: 'git@github.com:scylladb/siren-tests.git',
+                      credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
+                      branch: 'master')
+                  }
+                  dir('scylla-cluster-tests') {
+                      checkout scm
+                  }
+               }
+            }
+            stage('Create Cluster with siren-tests') {
+                steps {
+                        dir('siren-tests') {
+                            sh """
+                            #!/bin/bash
+                            set -xe
+
+                            export SIRENADA_BROWSER=chrome-headless
+
+                            source /opt/rh/rh-python35/enable
+                            # update the environment
+                            ~/.local/bin/pipenv --bare install
+
+                            export SIRENADA_REGION=eu-west-1
+                            export SIRENADA_NUMBER_OF_NODES=${params.n_db_nodes}
+                            export SIRENADA_CLUSTER_KEEP=true
+                            export SIRENADA_INSTANCE_TYPE=${params.db_instance_type}
+
+                            ~/.local/bin/pipenv run ./runtests.py --sct-conf
+                            """
+                        }
+                }
+            }
+            stage('Run SCT Test') {
+                steps {
+                    wrap([$class: 'BuildUser']) {
+                        dir('scylla-cluster-tests') {
+                            sh """
+                            #!/bin/bash
+                            set -xe
+                            env
+
+                            export SCT_CLUSTER_BACKEND=aws-siren
+                            export SCT_INTERNODE_COMM_PUBLIC=true
+                            export SCT_REGION_NAME=eu-west-1
+                            export SCT_CONFIG_FILES="['${pipelineParams.test_config}', '`realpath ../siren-tests/test_results/scylla_cloud.yaml`']"
+
+                            export SCT_FAILURE_POST_BEHAVIOR=${pipelineParams.params.get('post_behaviour', '')}
+                            export SCT_INSTANCE_PROVISION=${pipelineParams.params.get('provision_type', '')}
+                            export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+                            export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+
+                            echo "start test ......."
+                            ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend aws-siren --logdir /sct
+                            echo "end test ....."
+                           """
+                        }
+                    }
+                }
+            }
+        }
+        post {
```

Comment:
add ability to skip this step in order to keep cluster alive to debug issues

Reply:
Added